scalate-util/src/main/scala/org/fusesource/scalate/util/SourceMap.scala

```diff
@@ -253,7 +253,7 @@ class SourceMap {
    *                    that produced the <tt>smap</tt> to be embedded
    */
   def addSmap(smap: String, stratumName: String): Unit = {
```

Comment:
I’m afraid that you added an unnecessary +

Reply:
oops!
thanks for review

I fixed it.